resources/migrations/000_migrations.yaml

```diff
@@ -13990,6 +13990,21 @@ databaseChangeLog:
             constraintName: fk_action_creator_id
             nullable: false
 
+  - changeSet:
+      id: v46.00-074
+      author: metamben
+      comment: Added 0.46.0 -- increase precision of updated_at of report_card
+      changes:
+        - modifyDataType:
+            tableName: report_card
+            columnName: updated_at
+            newDataType: ${timestamp_type}
+      rollback:
+        - modifyDataType:
+            tableName: report_card
+            columnName: updated_at
+            newDataType: DATETIME
+
```

Comment:
Should this change be constrained to mysql and mariadb?

Reply:
I think it makes sense for Postgres and H2 too, since `${timestamp_type}` is `TIMESTAMP FOR TIME ZONE` for those, which is definitely preferrable to `DATETIME` for storing timestamps